tests/v4l2decode.cpp

```diff
@@ -715,7 +871,17 @@ int main(int argc, char** argv)
         }
     }
 #endif
-
+#if __ENABLE_WAYLAND__
+    struct v4l2_buffer buffer;
+    //queue buffs
+    for (uint32_t i = 0; i < outputQueueCapacity; i++) {
+        memset(&buffer, 0, sizeof(buffer));
+        buffer.type = V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE;
+        buffer.index = i;
```

Comment:
how about something like V4L2_MEMORY_DRIVERPTR? then we needn't make exception for the memory type elsewhere.